packages/formation-react/src/components/Pagination/Pagination.jsx

```diff
@@ -17,16 +18,30 @@ class Pagination extends Component {
 
   static defaultProps = {
     ariaLabelSuffix: '',
+    disableGATracking: false,
     maxPageListLength: 10,
   };
 
+  onPageSelect = (page, eventID) => {
+    // Propogate via the prop.
+    this.props.onPageSelect(page);
+
+    // Conditionally track the event.
+    if (window.dataLayer && !disableGATracking) {
```

Comment:
We will only attempt to track events if `window.dataLayer` exists and if they have not explicitly opted-out from tracking events with the `disableGATracking` prop.